src/Common/MemoryTracker.cpp

```diff
@@ -432,7 +438,7 @@ void MemoryTracker::reset()
 
 void MemoryTracker::setRSS(Int64 rss_, Int64 free_memory_in_allocator_arenas_)
 {
-    Int64 new_amount = rss_; // - free_memory_in_allocator_arenas_;
+    Int64 new_amount = rss_ - free_memory_in_allocator_arenas_;
```

Comment:
```suggestion
    Int64 new_amount = rss_;
```

Reply:
Or just
```
total_memory_tracker.amount.store(rss_, std::memory_order_relaxed);
```

Reply:
Then it will not solve the noisy message - https://github.com/ClickHouse/ClickHouse/blob/b8ee5600be1f8bb8eb3e3dcafdc3a467a09ff7c1/src/Interpreters/AsynchronousMetrics.cpp#L711-L722

Ok, `free_memory_in_allocator_arenas` can be removed here then.

Reply:
Yes, let's remove it